packages/main/src/Form.ts

```diff
@@ -301,6 +309,17 @@ class Form extends UI5Element {
 		});
 	}
 
+	setFastNavGroups() {
+		if (!this.hasGroupItems) {
+			this.setAttribute("data-sap-ui-fastnavgroup", "true");
```

Comment:
probably we need to remove attribute when the slot is changed and `hasGroupItem` returns true